src/Microsoft.Sbom.Api/Executors/IJsonSerializationStrategy.cs

```diff
@@ -18,15 +17,11 @@ internal interface IJsonSerializationStrategy
 
     public bool AddToExternalDocRefsSupportingConfig(IList<ISbomConfig> elementsSupportingConfigs, ISbomConfig config);
 
-    public void AddHeadersToSbom(ISbomConfigProvider sbomConfigs, ISbomConfig config)
-    {
-    }
-
-    public Task<List<FileValidationResult>> WriteJsonObjectsToSbomAsync(
-        ISbomConfig sbomConfig,
-        string spdxManifestVersion,
-        IJsonArrayGenerator<FileArrayGenerator> fileArrayGenerator,
-        IJsonArrayGenerator<PackageArrayGenerator> packageArrayGenerator,
-        IJsonArrayGenerator<RelationshipsArrayGenerator> relationshipsArrayGenerator,
-        IJsonArrayGenerator<ExternalDocumentReferenceGenerator> externalDocumentReferenceGenerator);
+    public void AddMetadataToSbom(ISbomConfigProvider sbomConfigs, ISbomConfig config);
+
+    public void StartGraphArray(IList<ManifestInfo> manifestInfosFromConfig, ISbomConfigProvider sbomConfigs);
```

Comment:
Probably worth including some comment of the intent for these methods--especially the ones that do nothing in 2.2